airflow-core/src/airflow/ui/src/i18n/locales/fr/admin.json

```
@@ -0,0 +1,162 @@
+  },
+  "formActions":{
+    "reset": "Réinitialiser",
+    "save": "Sauvegarder"
+  },
+  "plugins": {
+    "columns": {
+      "source": "Source"
+    },
+    "importError_one": "Erreur d'importation de plugin",
+    "importError_other": "Erreurs d'importation de plugins",
+    "searchPlaceholder": "Rechercher par fichier"
+  },
+  "pools": {
+    "add": "Ajouter une Pool",
+    "deferredSlotsIncluded": "Slots Différés Inclus",
+    "delete":{
+      "title": "Supprimer la Pool",
+      "warning": "Cela supprimera toutes les métadonnées liées à la pool et peut affecter les tâches utilisant cette pool."
+    },
+    "edit": "Modifier la Pool",
+    "form": {
+      "checkbox": "Check to include deferred tasks when calculating open pool slots",
+      "checkbox": "Cochez pour inclure les tâches différées lors du calcul des slots ouverts de la pool",
+      "description": "Description",
+      "includeDeferred": "Inclure les Tâches Différées",
+      "nameMaxLength": "Le nom peut contenir un maximum de 250 caractères",
+      "nameRequired": "Le nom est requis",
+      "slots": "Slots"
+    },
+    "noPoolsFound": "Aucune pool trouvée",
+    "pool_one": "Pool",
+    "pool_other": "Pools",
+    "searchPlaceholder": "Rechercher des Pools",
+    "sort": {
+      "asc": "Nom (A-Z)",
+      "desc": "Nom (Z-A)",
+      "placeholder": "Trier par"
+    }
+  },
+  "providers": {
+    "columns": {
+      "packageName": "Nom du Package",
+      "version": "Version"
+    }
+  },
+  "variables": {
+    "add": "Ajouter une Variable",
+    "columns": {
+      "isEncrypted": "Est Chiffrée"
+    },
+    "delete": {
+      "deleteVariable_one": "Supprimer 1 Variable",
+      "deleteVariable_other": "Supprimer {{count}} Variables",
+      "firstConfirmMessage_one": "Vous êtes sur le point de supprimer la variable suivante :",
+      "firstConfirmMessage_other": "Vous êtes sur le point de supprimer les variables suivantes :",
+      "title": "Supprimer la Variable",
+      "tooltip": "Supprimer les variables sélectionnées"
+    },
+    "edit":  "Modifier la Variable",
+    "export": "Exporter",
+    "exportTooltip": "Exporter les variables sélectionnées",
+    "form": {
+      "invalidJson": "JSON invalide",
+      "keyMaxLength": "La clé peut contenir un maximum de 250 caractères",
+      "keyRequired": "La clé est requise",
+      "valueRequired": "La valeur est requise"
+    },
+    "import": {
+      "button": "Importer",
+      "conflictResolution": "Sélectionner la Résolution de Conflit de Variable",
+      "errorParsingJsonFile": "Erreur lors de l'analyse du fichier JSON : Soumettez un fichier JSON contenant des variables (par exemple, {\"key\": \"value\", ...}).",
+      "options": {
+        "fail": {
+          "description": "Refuser l'importation si une variable avec la même clé existe déjà.",
+          "title": "Échouer"
+        },
+        "overwrite": {
+          "description": "Écrase la variable en cas de conflit.",
```

To have the same verb tenses as "fail" and "skip":
```suggestion
          "description": "Écraser la variable en cas de conflit.",
```

---

Thanks, fixed.